src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigrateBuildOptionsRule.cs

```diff
@@ -114,11 +114,22 @@ private AddPropertyTransform<CommonCompilerOptions>[] KeyFileTransforms
             "**/*.cs"
         };
 
+        private static bool IsPlainFileName(string fileName)
+        {
+            return !fileName.Contains('/') && !fileName.Contains('\\');
```

Comment:
Yeah in that case let's just keep it like it is. This will still cover many common cases I think.